### DISCUSSION TITLE
Volume control of USB Audio is binary

### DISCUSSION THREAD

**Discussion by solsticedhiver** [jcorporation/myMPD#1393] [Q&A] [answered]
Testing the latest mpd from git 0.24 (v0.23.16-1610-gb1677bf79+), I can now only control the USB Audio device I am using with the alsa plugin, being either at full volume or quiet.

Say it is 0, if I increase the volume to 5% in mympd, the volume is set at max. I can check that in alsamixer.

Is it mympd that can't talk to the latest mpd, or is a bug(?) in mpd, or something else in between, may be at the lasa level, of the mpd alsa config, that has changed somehow. I have not dig into the latest.

**Comment by jcorporation** [comment]
Test it with an other client like mpc.

**Reply by solsticedhiver** [reply]
It's the same behavior with mpc.

**Reply by solsticedhiver** [reply]
I tried using the pipewire output of mpd, but that does not work. no sound. any know how to specify the target option for pipewire settings in mpd.conf ?

https://mpd.readthedocs.io/en/latest/plugins.html#pipewire

I tried the various thing given by `pw-cli ls Node` like  node.name, device.id client.id @DEFAULT_AUDOO_SINK@ or even not specifying ay target. Still no  luck

**Reply by jcorporation** [reply, marked as answer]
As it is no myMPD issue, you should ask this question in the [MPD discussions](https://github.com/MusicPlayerDaemon/MPD/discussions). You have a better chance of getting an answer there.
